src/osdc/Objecter.cc

```diff
@@ -2763,12 +2763,8 @@ int Objecter::_calc_target(op_target_t *t, Connection *con, bool any_change)
   pg_t actual_pgid(actual_ps, pgid.pool());
   pg_mapping_t pg_mapping;
   pg_mapping.epoch = osdmap->get_epoch();
-  if (lookup_pg_mapping(actual_pgid, &pg_mapping)) {
-    up = pg_mapping.up;
-    up_primary = pg_mapping.up_primary;
-    acting = pg_mapping.acting;
-    acting_primary = pg_mapping.acting_primary;
-  } else {
+  if (!lookup_pg_mapping(
+          actual_pgid, &up, &up_primary, &acting, &acting_primary)) {
```

Comment:
Need to pass `epoch` to `lookup_pg_mapping()`.  And don't wrap after the open paren.